enterprise/management/commands/email_drip_for_missing_dsc_records.py

```diff
@@ -180,7 +180,9 @@ def handle(self, *args, **options):
                 course_accessed = True
 
             # Emit the Segment event which will be used by Braze to send the email
-            if isinstance(consent, ProxyDataSharingConsent) and course_accessed:
+            if (isinstance(consent, ProxyDataSharingConsent) and
+                    course_accessed and
+                    enterprise_customer.enable_data_sharing_consent):
```

Comment:
@irfanuddinahmad  I think `enterprise_customer.enable_data_sharing_consent` check should be at very start of this management command. If `DSC` is not enabled for a customer then we should return as early as possible.

Reply:
The volume of the transactions is too small for this kind of optimisation.

Reply:
The volume of the transactions is too small for this optimisation.